scipy/fft/_pocketfft/basic.py

```diff
@@ -23,8 +23,8 @@ def c2c(forward, x, n=None, axis=-1, norm=None, overwrite_x=False,
         tmp, copied = _fix_shape_1d(tmp, n, axis)
         overwrite_x = overwrite_x or copied
     elif tmp.shape[axis] < 1:
-        raise ValueError("invalid number of data points ({}) specified"
-                         .format(tmp.shape[axis]))
+        raise ValueError(f"invalid number of data points ({tmp.shape[axis]}) specified"
+                         )
```

Comment:
Ugh I guess there are several of these. I guess we can let it go.
```suggestion
        message = f"invalid number of data points ({tmp.shape[axis]}) specified"
        raise ValueError(message)
```